src/metatrain/experimental/gap/model.py

```diff
@@ -213,13 +259,18 @@ def forward(
         output_key = list(outputs.keys())[0]
         energies = self._subset_of_regressors_torch(soap_features)
         return_dict = {output_key: energies}
+        print("SOAP ENERGIES", energies.block().values)
 
         if not self.training:
             # at evaluation, we also add the additive contributions
             for additive_model in self.additive_models:
                 additive_contributions = additive_model(
                     systems, outputs, selected_atoms
                 )
+                print(
```

Comment:
is this writing every time you do an evaluation?

Reply:
Sorry, this was a leftover from the testing. Removed